features/facebookImport/src/components/loading/loading.jsx

```diff
@@ -6,10 +6,9 @@ import "./loading.css";
 
 const Loading = ({ message }) => (
     <div className="loading">
-        <div>
-            <p>{message} ...</p>
-            <p>{i18n.t("common:loading.closing")}</p>
-        </div>
+        <img src="./images/ajax-loader.gif" />
```

Comment:
Seems like you're not adding the image file here! I'd also rename it to `spinner.gif` or something like that, no strong preference on the name though.